test/fixtures/basic/server/routes/proxy.ts

```diff
@@ -0,0 +1,3 @@
+export default defineEventHandler(async () => {
+  return await $fetch<string>('/')
```

Comment:
As best practice, we should use `event.$fetch` to preserve provider context

Reply:
interesting bug - it does not support a generic 🤔